src/root/common.tt

```diff
@@ -476,6 +476,9 @@ BLOCK renderEvals %]
             ELSE %]
              -
             [% END %]
+            [% IF eval.errormsg %]
+              <span class="label label-warning">Eval Errors</span></td>
```

Comment:
I need to spin up an hydra instance, cause eval errors, but that `</td>` at first glance looks oddly unmatched.

Reply:
Whoops!

Reply:
Fixed up!